.travis.rosinstall

```diff
@@ -1,3 +0,0 @@
-- git:
-    uri: https://github.com/ros-industrial/abb.git
-    local-name: ros-industrial/abb
```

Comment:
The concept is needed, but I just put it straight into the .travis.yml. This essentially uses the rosinstall method to download the abb repo.
```
global:
    - ROS_DISTRO=kinetic
    - UPSTREAM_WORKSPACE=file
    - ROSINSTALL_FILENAME=abb_experimental.rosinstall
```